src/components/ui/form/button/index.jsx

```diff
@@ -0,0 +1,18 @@
+import PropTypes from "prop-types";
+import "./style.scss";
+
+function Button({ children, className, type = "button", ...rest }) {
+    return (
+        <button type={type} className={`btn ${className}`} {...rest}>
+            {children}
+        </button>
+    );
+}
+
+Button.propTypes = {
+    children: PropTypes.node.isRequired || PropTypes.string.isRequired,
```

Comment:
What could we expect inside node?

Reply:
I've changed it and defined it with `PropTypes.element`.